### PR TITLE
Feature/security update mongoose

### PR DESCRIPTION
**Pull request by jon-shipley**
Update out of date packages.

### CODE REVIEW

File: admin/app.js [hunk 1]
```diff
@@ -45,17 +41,26 @@ if (unsetVars.length > 0) {
...
+  keepAlive: true,
+  reconnectTries: 120,
+  // set the delay between every retry (milliseconds)
+  reconnectInterval: 1000,
+  useMongoClient: true
 })
+
 autoIncrement.initialize(mongoose.connection)
 
+if (process.env.NODE_ENV !== 'production') {
```
Comment (GuyHarwood) [inline admin/app.js]: we already have this on line 45!

File: admin/spec/models/completed-checks.spec.js [hunk 2]
```diff
@@ -24,7 +24,6 @@ describe('CompletedChecks model', () => {
     check.receivedByServerAt = undefined
     check.validate(err => {
       expect(err).toBeDefined()
-      expect(err.message).toBe('completedChecks validation failed')
```
Comment (GuyHarwood) [inline admin/spec/models/completed-checks.spec.js]: how come this was removed?  seems out of context 🤔

**Comment by GuyHarwood** [comment]
@jon-shipley are we able to update the handlebars dependency in the SPA?  as we are on a low version and github keeps reporting a vulnerability...

https://github.com/DFEAGILEDEVOPS/MTC/network/dependencies#606634

File: admin/spec/models/completed-checks.spec.js (hunk 2 above)
Comment (jon-shipley) [reply admin/spec/models/completed-checks.spec.js]: The expectation was brittle.  That is not an error message out app produces - rather it is one produced from the mongoose library, which changed when I upgraded the mongoose package to latest.

It seems sufficient that it checks that there is an error caused by the `receivedByServerAt` field on the next line.

**Comment by jon-shipley** [comment]
The handlebars vulnerability is fixed in commit 6fa110c.

File: admin/app.js (hunk 1 above)
Comment (jon-shipley) [reply admin/app.js]: Oops! Now fixed.

File: admin/spec/models/completed-checks.spec.js (hunk 2 above)
Comment (GuyHarwood) [reply admin/spec/models/completed-checks.spec.js]: makes sense to me 👍

**Pull request merged**